components/webstories/components/ShareComponent.tsx

```diff
@@ -80,6 +81,15 @@ export default function CustomizedDialogs({handlingPauseBehindScenes}:{handlingP
     }
   }, []);
 
+  React.useEffect(() => {
+    if (typeof window !== "undefined") {
+      setWindowWidth(window.innerWidth);
```

Comment:
this piece of code is being used at multiple. Can we implement context for height and width and use in other components